kratos/python_scripts/assign_initial_state_process.py

```diff
@@ -0,0 +1,125 @@
+# Importing the Kratos Library
+import KratosMultiphysics
+
+
+def Factory(settings, Model):
+    if not isinstance(settings, KratosMultiphysics.Parameters):
+        raise Exception(
+            "expected input shall be a Parameters object, encapsulating a json string"
+        )
+    return SetInitialStateProcess(Model, settings["Parameters"])
+
+
+# All the processes python processes should be derived from "Process"
+
+
+def components_to_string(vect):
+    v = []
+    for c in vect:
+        if c.IsNumber():
+            s = str(c.GetDouble())
+        elif c.IsString():
+            s = c.GetString()
+        else:
+            msg = "SetInitialStateProcess: Vector component must be scalar or string"
+            raise Exception(msg)
+        function = KratosMultiphysics.PythonGenericFunctionUtility(s)
+        if function.DependsOnSpace():
+            msg = "SetInitialStateProcess: 'x', 'y' and 'z' variables not supported yet"
+            raise Exception(msg)
+        v.append(function)
+    return v
```

Comment:
what does this do?
Cannot it be done in the same was as the other processes?

Reply:
@marandra

Reply:
> what does this do?

The `components_ to_string` function receives a vector as written by the user, and converts its components to strings, and in turn to functions. In this way, the user can pass a vector like [0.0, "1.2", "3.4 + t"] and it will be stored as [f(0), f(1.2), f(3.4+t)).
(I realize that it should be called something like "components_to _functions" and that its missing the docstrings.)

> Cannot it be done in the same was as the other processes?

I was looking for examples on how implement this functionality, I only found it used in the  `assign_scalar_variable_process.py` process, but it looked to me an ad-hoc implementation. If there is a recommended way to implement it, I will happily do it.

There are some more missing details regarding the process, like adapting it to 2D, and converting to functions the components of the matrix too. I'll push another commit with it.